dictionaries/CallMap.php

```diff
@@ -10273,7 +10273,7 @@
 'preg_filter' => ['null|string|string[]', 'pattern'=>'mixed', 'replacement'=>'mixed', 'subject'=>'mixed', 'limit='=>'int', '&w_count='=>'int'],
 'preg_grep' => ['array|false', 'pattern'=>'string', 'array'=>'array', 'flags='=>'int'],
 'preg_last_error' => ['int'],
-'preg_match' => ['int|false', 'pattern'=>'string', 'subject'=>'string', '&w_matches='=>'string[]', 'flags='=>'0|', 'offset='=>'int'],
+'preg_match' => ['int|false', 'pattern'=>'string', 'subject'=>'string', '&w_matches='=>'string[]', 'flags='=>'0', 'offset='=>'int'],
```

Comment:
Why flags are 0 here?

Reply:
Was already there.
But I guess we could probably simply delete this line and and just use the ``preg_match\'1`` variant?!

Reply:
Ah, I missed the overload. In that case it's fine to leave it in the way you changed it to - with no flags `$matches` will be single-dimensional array of strings.